csharp/Adapter/Microsoft.Spark.CSharp/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.Spark.CSharp.Adapter")]
+[assembly: AssemblyDescription("")]
```

Comment:
Do we want to make use of this tag and provide a short description to the 3 binaries we release?